rector.php

```diff
@@ -61,4 +61,21 @@
     $rectorConfig->rule(\Rector\DeadCode\Rector\For_\RemoveDeadIfForeachForRector::class);
     $rectorConfig->rule(\Rector\DeadCode\Rector\If_\RemoveDeadInstanceOfRector::class);
     $rectorConfig->rule(\Rector\Symfony\Rector\MethodCall\ContainerGetToConstructorInjectionRector::class);
+
+    $rectorConfig->rule(\Rector\Doctrine\Rector\Property\ChangeBigIntEntityPropertyToIntTypeRector::class);
```

Comment:
I did not use rule sets because there are some rules that will not work for us at the moment or outside the scope of this PR. once we merge this PR I will move rector to sets. I have in my mind

Reply:
Understood. That's what I did with the dead code rules too. Next time please create one PR per new rule to make the reviews smoother.